_data/other.yml

```diff
@@ -285,9 +285,10 @@ websites:
     - name: Indeed
       url: https://www.indeed.com/
       img: indeed.png
-      tfa: No
-      twitter: indeed
-      facebook: Indeed
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      doc: https://support.indeed.com/hc/en-us/articles/360000244123-What-is-Two-Step-Verification-
```

Comment:
Could you just do a quick change and make the doc url to `https://support.indeed.com/hc/en-us/articles/360000244123`?
I think that would help if they decided to change the tile of the page which sometimes happens.